style.css

```diff
@@ -1,3 +1,9 @@
+body {
+  background-image: url("./assets/header-mobile-view.png");
+  background-repeat: no-repeat;
+  background-size: 100% auto;
+}
+
 .navbar {
```

Comment:
- [ ] As your previous reviewer pointed out, your design should look as much as possible to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=T8hTVtXsVQcXir8T-0) 🫣🫣🫣 Note that having a beautiful design would increase your chance of landing your dream job 🤠
I think the issue here is the <code>background-color</code>, in the Figma template there is no background color, so please remove that 🙏🏽

Reply:
@rica213 from where from these 
 .navbar {
  display: flex;
  justify-content: space-between;
  align-items: center;
  height: 50px;
  background-color: hsl(225, 17%, 95%);
  color: #3b2df8;
}

Reply:
@melkamu372 , you can just remove this line <code>background-color: hsl(225, 17%, 95%);</code> 😁